HeterogeneousCore/SonicTriton/interface/TritonData.h

```diff
@@ -58,6 +61,17 @@ class TritonData {
   //default to dims if shape isn't filled
   int64_t sizeShape() const { return variableDims_ ? dimProduct(shape_) : sizeDims(); }
 
+  std::string defaultConverter() const {
+    std::string base = "StandardConverter";
+    if (dname_ == "INT64") {
+      return "Int64"+base;
+    }
+    else if (dname_ == "FP32") {
+      return "Float"+base;
+    }
+    return "";
```

Comment:
given the way this is currently used in the `TritonClient` constructor, it might be better for it to throw if it can't find a default converter (since the constructor already checked if a non-default converter was requested)
also, I'd recommend putting this in the private section (`TritonClient` is a friend class, so it can still call the function)